### ISSUE TITLE
Feature configuration pages 

### ISSUE THREAD

**Issue by dpmurphy** [HomeSeer/Plugin-SDK#199]
**Environment and System Config**
- OS: [Windows]
- Language: [C#]
- IDE: [VS2019]
- HomeSeer Version: [v4.1.16.0]
- PluginSDK Version: [v1.0.5.1]

I would like to propose that you consider having a "feature" configuration page. Selecting a feature currently the config page is passed the root device id not the feature id. It appears you have decided the root device configures everything. Where as the MySensors plugin has multiple features most of these features currently has specific configuration parameters in HS3.

It has been suggested that the user could click on the feature, then select the config page, which has a drop down with all the features, select the required features, update the config page and save the updated feature configuration. BUT this just seems so ass backwards an approach...I'm really hoping you'll make a change.

**Priority**
How important is this to you on a scale of 1-5?
4. Critical

**Comment by alexbk66** [comment]
There are **two duplicate** open issues already
https://github.com/HomeSeer/Plugin-SDK/issues/86
https://github.com/HomeSeer/Plugin-SDK/issues/157

**Comment by dpmurphy** [comment]
Ooops.  So HST will close, probably not going to fix the issue.  As those open issues have some big developer names attached and no progress.....darn I don’t get it.

**Comment by alexbk66** [comment]
No, it's good you also open another issue - it shows that it's not only me.

But your name is there too https://github.com/HomeSeer/Plugin-SDK/issues/86#issuecomment-801287720
